src/test/resources/features/checkbox.feature

```diff
@@ -1,7 +1,9 @@
 Feature: Checkboxes
   Scenario: User can click on checkboxes
     Given User is on the checkbox page
-    When User clicks checkbox one "1"
-    And User clicks checkbox two "2"
-    Then Checkbox one is checked "1"
-    And Checkbox two is unchecked "2"
+    When User clicks checkbox 1
+    And User clicks checkbox 2
+    Then Checkbox is checked 1
+    And Checkbox is unchecked 2
+    #could make a data table if there were more than 2 checkboxes to test here
```

Comment:
you can still make datatable, even if you only have one option! It is more elegant and scalable - once you add more options, there is no need for new code

Reply:
Data tables added.